style.css

```diff
@@ -1,8 +1,9 @@
+@import url('https://fonts.googleapis.com/css2?family=Cabin&family=Lato&family=Telex');

```

Comment:
All fonts that were trialed were used in the final design, except for Merriweather